src/components/main/StaffList.js

```diff
@@ -4,12 +4,56 @@ import './StaffList.css';
 
 class StaffList extends React.Component {
    
-  
+    listAdd = (index,sf) =>{
+        let quantity = parseInt(this.props.staffs[index].quantity)
+        let shiftTotal = parseInt(this.props.staffs[index].shiftTotal)
+        let shift = parseInt(this.props.staffs[index].shift)
+        quantity = quantity + 1
+
+        if (shift == 12) {
```

Comment:
You have compile warnings in your code. It should be 'if (shift === 12)'. Before you do a PR, all compile warnings should be resolved. We agreed that all compile warnings should be resolved in a meeting on October 19th and we recorded it in our meeting minutes, if you need to review that.